OpenEYE TK version checking is causing test failures in `alchemy-prep`

```
FAILED asapdiscovery-alchemy/asapdiscovery/alchemy/tests/test_alchemy_prep.py::test_prep_workflow[Strict-smarts] - AssertionError: assert {'OpenEyeCons...a': 20231212}} == {'OpenEyeCons...a': 20230910}}
  Differing items:
  {'StereoExpander': {'expander': {'expander_type': 'StereoExpander', 'stereo_expand_defined': False}, 'oechem': 20231212, 'omega': 20231212}} != {'StereoExpander': {'expander': {'expander_type': 'StereoExpander', 'stereo_expand_defined': False}, 'oechem': 20230910, 'omega': 20230910}}
  {'OpenEyeConstrainedPoseGenerator': {'oechem': 20231212, 'oedocking': 20231212, 'oeff': 20231212, 'oeomega': 20231212}} != {'OpenEyeConstrainedPoseGenerator': {'oechem': 20230910, 'oedocking': 20230910, 'oeff': 20230910, 'oeomega': 20230910}}
  Full diff:
    {
  -  'OpenEyeConstrainedPoseGenerator': {'oechem': 20230910,
  ?                                                    -- ^
  +  'OpenEyeConstrainedPoseGenerator': {'oechem': 20231212,
  ?                                                     ^^^
  -                                      'oedocking': 20230910,
  ?                                                       -- ^
  +                                      'oedocking': 20231212,
  ?                                                        ^^^
  -                                      'oeff': 20230910,
  ?                                                  -- ^
  +                                      'oeff': 20231212,
  ?                                                   ^^^
  -                                      'oeomega': 20230910},
  ?                                                     -- ^
  +                                      'oeomega': 20231212},
  ?                                                      ^^^
  
  ```

--- a/asapdiscovery-alchemy/asapdiscovery/alchemy/tests/test_alchemy_prep.py
+++ b/asapdiscovery-alchemy/asapdiscovery/alchemy/tests/test_alchemy_prep.py
@@ -42,21 +42,13 @@ def test_prep_workflow(strict_stereo, core_smarts, failed, mac1_complex):
     )
 
     assert len(alchemy_dataset.input_ligands) == 1
-    assert alchemy_dataset.provenance == {
-        "OpenEyeConstrainedPoseGenerator": {
-            "oechem": 20230910,
-            "oedocking": 20230910,
-            "oeff": 20230910,
-            "oeomega": 20230910,
-        },
-        "StereoExpander": {
-            "expander": {
-                "expander_type": "StereoExpander",
-                "stereo_expand_defined": False,
-            },
-            "oechem": 20230910,
-            "omega": 20230910,
-        },
+    assert alchemy_dataset.provenance.keys() == [
+        "OpenEyeConstrainedPoseGenerator",
+        "StereoExpander",
+    ]
+    assert alchemy_dataset.provenance["StereoExpander"]["expander"] == {
+        "expander_type": "StereoExpander",
+        "stereo_expand_defined": False,
     }
     if failed:
         assert len(alchemy_dataset.posed_ligands) == 0

--- a/asapdiscovery-alchemy/asapdiscovery/alchemy/tests/test_alchemy_prep.py
+++ b/asapdiscovery-alchemy/asapdiscovery/alchemy/tests/test_alchemy_prep.py
@@ -42,10 +42,10 @@ def test_prep_workflow(strict_stereo, core_smarts, failed, mac1_complex):
     )
 
     assert len(alchemy_dataset.input_ligands) == 1
-    assert list(alchemy_dataset.provenance.keys()) == [
+    assert set(alchemy_dataset.provenance.keys()) == {
         "OpenEyeConstrainedPoseGenerator",
         "StereoExpander",
-    ]
+    }
     assert alchemy_dataset.provenance["StereoExpander"]["expander"] == {
         "expander_type": "StereoExpander",
         "stereo_expand_defined": False,

--- a/asapdiscovery-alchemy/asapdiscovery/alchemy/tests/test_alchemy_prep.py
+++ b/asapdiscovery-alchemy/asapdiscovery/alchemy/tests/test_alchemy_prep.py
@@ -42,7 +42,7 @@ def test_prep_workflow(strict_stereo, core_smarts, failed, mac1_complex):
     )
 
     assert len(alchemy_dataset.input_ligands) == 1
-    assert alchemy_dataset.provenance.keys() == [
+    assert list(alchemy_dataset.provenance.keys()) == [
         "OpenEyeConstrainedPoseGenerator",
         "StereoExpander",
     ]

